notes/authenc.tex

```diff
@@ -73,6 +73,7 @@ \section{Authenticated Encryption}
 \enm
 Adversary $\advB$ makes at most $q$ 
 queries and runs in time that of $\advA$.
+\label{theorem:ror-cpa}
```

Comment:
this theorem was already referenced in randomized encryption, so should delete here

Reply:
you should also move the security game and advantage definitions for `ror-cca` and `ctxt` in 9.2, when you talk about them